doc/source/whatsnew/v0.24.0.txt

```diff
@@ -182,6 +182,7 @@ Other Enhancements
 - :func:`to_timedelta` now supports iso-formated timedelta strings (:issue:`21877`)
 - :class:`Series` and :class:`DataFrame` now support :class:`Iterable` in constructor (:issue:`2193`)
 - :class:`DatetimeIndex` gained :attr:`DatetimeIndex.timetz` attribute. Returns local time with timezone information. (:issue:`21358`)
+- :meth:`DataFrame.corrwith` now supports Spearman's rank correlation as well as Kendall's tau. (:issue:`21925`)
```

Comment:
This makes is sound like Knedall's tau was the previously supported method. Maybe "... supports Spearman's rank correlation and Kendall's tau for `method`"

Reply:
Good point, will make that change.